stable/minio/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Minio is a high performance distributed object storage server, designed for large-scale private cloud infrastructure.
 name: minio
-version: 2.1.0
-appVersion: RELEASE.2018-11-30T03-56-59Z
+version: 2.1.1
```

Comment:
I suggest this to be `2.2.0` at least since this is not bug fixing and have some new features and maybe some breaking upgrade.